README.md

```diff
@@ -55,7 +55,7 @@ Below is a table with supported platforms, python versions and architectures.
 
 OS | Py2 | Py3 | 32bit | 64bit | Requirements
 --- | --- | --- | --- | --- | ---
-Windows | 2.7 | 3.4 / 3.5 / 3.6 / 3.7 / 3.8 / 3.9 | Yes | Yes | Windows 7+ (Note that Python 3.9 supports Windows 8.1+)
+Windows | 2.7 | 3.4 / 3.5 / 3.6 / 3.7 / 3.8 / 3.9 / 3.10 / 3.11 | Yes | Yes | Windows 7+ (Note that Python 3.9 supports Windows 8.1+)
```

Comment:
That OS table in Install section which has instructions to use `pip install cefpython3==66.1` which doesn't yet support 3.10 and 3.11.